deny.toml

```diff
@@ -103,6 +103,11 @@ exceptions = [
     # this is a LGPL like license in the CLI
     # since this is an application not a library people would link to it should be fine
     { allow = ["EPL-2.0"], name = "colored_json" },
+
+    # these are needed by cbindgen and its dependancies
+    # should be revied more fully before release
```

Comment:
Does this mean they are build time only dependencies? If not we would need to discuss whether we mind that cgindgen is distributed under the MPL with the "Incompatible with secondary licenses" notice, which apparantly makes it incompatible with the GPL. I don't know whether this matters.

Reply:
> Does this mean they are build time only dependencies?

@alexjg, `cbindgen` doesn't require it's own runtime support library so it's a build-time-only dependency.

Reply:
Ah okay, then I think this should be fine.